OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -13899,7 +13899,7 @@ algorithm
 
       // Save value
       if stringEqual(tmpName, "s") then
-        if not stringEqual(tmpValue, "euler") and not stringEqual(tmpValue, "cvode") then
+        if not stringEqual(tmpValue, "euler") and not stringEqual(tmpValue, "cvode") and not stringEqual(tmpValue, "ida") then  //add ida by twxin 2024.3.26
```

Comment:
Git will track who changed this line, so no need to add your name to every location you changed.